main.go

```diff
@@ -230,8 +233,10 @@ func (k *Krud) update(h *Webhook) error {
 	if err != nil {
 		return err
 	}
-	ns := api.NamespaceDefault
-	rcs := client.ReplicationControllers(ns)
+	if k.Namespace == "" {
```

Comment:
My other comments have suggested setting the code's default to api.NamespaceDefault. Those changes would make this block unneeded, since the namespace should always be set by this point.